src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/magicnumber/InputMagicNumberIntMethodAnnotation.java

```diff
@@ -1,5 +1,7 @@
 package com.puppycrawl.tools.checkstyle.checks.coding.magicnumber;
 
+
```

Comment:
Are these blank lines intentional / important?

Reply:
@esilkensen sorry, my bad. I think it occurred because I was intending to add something but found its not required

Reply:
I have deleted those lines and pushed the code again